src/main/java/mike/Storage.java

```diff
@@ -0,0 +1,60 @@
+package mike;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import exception.MikeCommandExecutionException;
+
+public class Storage {
+    private File listFile;
+
+    public Storage(String filePath) {
+        this.listFile = new File(filePath);
+    }
+
+    /**
+     * Update saved tasklist file after command execution
+     * Create file if it does not exist
+     * @param taskList taskList to be saved
+     * @throws MikeCommandExecutionException if tasklist cannot be saved to file
+     */
+    public void updateListFile(TaskList taskList) throws MikeCommandExecutionException {
```

Comment:
Maybe updateListInFile is more appropriate and more similar to your other method readListFromFile.